docs/release-process.md

```diff
@@ -4,17 +4,39 @@
 We use [cargo-release](https://crates.io/crates/cargo-release) to simplify the release process.
 (We rely on v0.22 or later because it has support for workspaces. Install this with
 `cargo install cargo-release`, not to be confused with the different `cargo install release`!)
-It's not (yet) quite an ideal fit for our workflow, but it helps! Steps:
+It's not (yet) quite an ideal fit for our workflow, but it helps!
 
-1. Take a look over `CHANGELOG.md` and make sure the "unreleased" section accurately reflects the
-   contents of the new release.
-   * Anything that could cause a consumer project to behave differently if it upgraded
-     to the new version, should be called out as a breaking change and should trigger
-     a minor version bump (we're below `v1.0` for semver purposes).
+We use a separate version number for `uniffi` compared to all the other crates:
+
+  - The `uniffi_*` crates are intended for either internal use or by external bindings generators.
+    These crates get breaking version bumps regularly.
+  - `uniffi` acts as a frontend to the other crates and re-exports the top-level UniFFI
+    functionality.  `uniffi` gets breaking version bumps less often than the other crates (see below
+    for details).
+
+Steps:
+
+1. Take a look over `CHANGELOG.md` and make sure the "unreleased" section lists all changes for
+   the release.
+   * Anything that affects any UniFFI consumers should be listed, this includes consumers that
+     use UniFFI to generate their scaffolding/bindings, external bindings generators, etc.
+
+1. Take a look over `uniffi/CHANGELOG.md` and make sure the "unreleased" section lists changes for
+   the release that affect users `uniffi` of the UniFFI crate.
+   * This should be a copy of the items from the top-level `CHANGELOG.md` that affects
+     scaffolding/bindings generation.
+   * Note that breaking changes for a particular `uniffi_*` crate are not necessarily breaking for
+     the `uniffi` crate.  See `./uniffi-versioning.md` for a discussion of this.
+
+1. Decide on a new version number for `uniffi` crate.  Since we are pre-`1.0`, if there are breaking
+   changes then this should be a minor version bump, otherwise a patch version bump.
+
+1. Decide on a new version number for the other `uniffi_*` crates.
```

Comment:
Ah, for some reason I was assuming we'd only bump version numbers for sub-crates which changed, so those numbers would start diverging - but given the above, that's not correct, right? I wonder if that's worth spelling out somewhere?